src/app/heroes/heroes.component.ts

```diff
@@ -11,6 +11,11 @@ import { HeroService } from '../hero.service';
 export class HeroesComponent implements OnInit {
 
   heroes: Hero[] = [];
+  delete(hero: Hero): void {
+    this.heroes = this.heroes.filter(h => h !== hero);
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/infinity-arc/Kobus-tour-of-heroes/pullRequest?prid=7840995)
```suggestion
    this.heroes = this.heroes.filter((h) => h !== hero);
```